static_src/util/cf_api.js

```diff
@@ -403,7 +403,22 @@ export default {
   deleteOrgUserPermissions(userGuid, orgGuid, apiKey) {
     return http.delete(`${APIV}/organizations/${orgGuid}/${apiKey}/${userGuid}`)
       .then((res) => res.response
-    );
+    ).catch(res => {
+      const err = parseError(res);
+      const errorConditions = (res &&
+                               res.response &&
+                               res.response.status === 400 &&
+                               res.response.data.error_code === 'CF-AssociationNotEmpty'
+                              );
+      if (errorConditions) {
+        const description = 'This user can\'t be removed because they still have a space ' +
```

Comment:
you don't have a test that calls `deleteOrgUserPermissions` and verifies that `createUserSpaceAssociationNotification` is called with the description